test/functions/casts/stringToUint.chpl

```diff
@@ -0,0 +1,16 @@
+var binL = 0b111;
+var binS = "0b111": uint;
+
+var octL = 0o777;
+var octS = "0o777": uint;
+
+var decL = 999;
+var decS = "999": uint;
+
+var hexL = 0xFFF;
+var hexS = "0xFFF": uint;
+
+assert(binL == binS);
+assert(octL == octS);
+assert(decL == decS);
+assert(hexL == hexS);
```

Comment:
We like to include a newline at the end of the file.

Reply:
Please also add the example from issue #12616 to this test.

Reply:
okay, will add newline at the end of file

> We like to include a newline at the end of the file.
Example Written in #12616 and the example which I've included, both are almost similar. if my test will pass then example in #12616 will also pass. Should i still add the one which is in #12616 ? 
> Please also add the example from issue #12616 to this test.